yarn.lock

```diff
@@ -16782,6 +16782,27 @@ react-inspector@^2.3.0, react-inspector@^2.3.1:
     is-dom "^1.0.9"
     prop-types "^15.6.1"
 
+react-instantsearch-dom-maps@6.10.2:
+  version "6.10.2"
+  resolved "https://registry.yarnpkg.com/react-instantsearch-dom-maps/-/react-instantsearch-dom-maps-6.10.2.tgz#2666e8fb5e46716057cb4efffa22394a87cbd4d9"
+  integrity sha512-c7AbzVZ6eeH960A23ksLgltrMmNk+CyGalq6imzfeKfKnjPuyORYlqa9cyDZnUHET+nOxlVBYj1TS/g2ARU0bQ==
+  dependencies:
+    "@babel/runtime" "^7.1.2"
+    prop-types "^15.6.2"
+    scriptjs "^2.5.8"
+
+react-instantsearch-dom@6.10.2:
+  version "6.10.2"
+  resolved "https://registry.yarnpkg.com/react-instantsearch-dom/-/react-instantsearch-dom-6.10.2.tgz#c969f84243fed8ddd9eebe440042a3dd32ef5c86"
+  integrity sha512-5u5t5lvydgTq218JmbOv7BZqFmvGcpgQTEi/02M/T0BLjToDu8bMItpXXQHwWNlOidJl/M+iphybxkxAvJukFQ==
+  dependencies:
+    "@babel/runtime" "^7.1.2"
+    algoliasearch-helper "^3.4.3"
+    classnames "^2.2.5"
+    prop-types "^15.6.2"
+    react-fast-compare "^3.0.0"
+    react-instantsearch-core "^6.10.2"
+
```

Comment:
this seems wrong?

Reply:
Yep, I don't have it in master

Reply:
It was removed here: https://github.com/algolia/react-instantsearch/commit/fa815f970dea19f96c8275a1c08061f34c4a50ad#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deL16785-L16805, that's it?

Reply:
hmm it's because of examples using the old version